torch/csrc/jit/mobile/import.cpp

```diff
@@ -280,7 +278,7 @@ void BytecodeDeserializer::parseMethods(
   TORCH_CHECK(vals.size() > 0, "Bytecode has no elements. ");
   // Initialized with the version number when kProducedBytecodeVersion was
   // introduced. The old models (some of them already in production) without
-  // version number don't have to be re-generated.
+  // version number are seen as version 3 (deprecated).
   int64_t model_version = 0x3L;
```

Comment:
I'd suggest making it a constant itself (e.g. default legacy version or something like that).
Is this the only place that will use this constant?
Otherwise maybe move it to caffe2/serialize/versions.h ?

Reply:
Good catch. I'll add const here. It's the only place that it's used.

Reply:
Good catch. I'll add const here. It's the only place that it's used.